web/client/epics/dashboard.js

```diff
@@ -180,6 +183,28 @@ export const saveDashboard = action$ => action$
             )
     );
 
+export const exportDashboard = action$ => action$
+    .ofType(DASHBOARD_EXPORT)
+    .switchMap(({originalData, resource, services}) =>
+        Rx.Observable.of([JSON.stringify({originalData, resource, services}), 'dashboard.json', 'application/json'])
```

Comment:
I saw that it's called originalData in the redux state but maybe we could use data for the json file.
Proposal: change `originalData` to `data`.